ci-operator/config/openshift/windows-machine-config-operator/openshift-windows-machine-config-operator-community-4.12__okd.yaml

```diff
@@ -53,7 +53,7 @@ tests:
   container:
     from: src
 - as: aws-e2e-operator
-  cron: 0 */4 * * *
+  cron: 0 */4 * * 1-5
```

Comment:
Why do we want to run it every 4 hours? Just once a day is enough. The more I think about it maybe running it once a day on Tue and Friday will suffice.

Reply:
I want to run it once a day eventually - I did it every four hours for now so I didn't have to wait a whole day to open the next iterative PR

Reply:
Ah I see. I didn't realize you were still iterating.